server/api/db/sqldb/models/models_mysql.py

```diff
@@ -702,6 +702,29 @@ def full_object(self):
         def full_object(self, value):
             self._full_object = json.dumps(value, default=str)
 
+    class AlertTemplate(Base, mlrun.utils.db.BaseModel):
+        __tablename__ = "alert_templates"
+        __table_args__ = (UniqueConstraint("name", name="_alert_templates_uc"),)
+
+        id = Column(Integer, primary_key=True)
+        name = Column(
+            String(255, collation=SQLCollationUtil.collation()), nullable=False
+        )
+
+        _full_object = Column("object", JSON)
```

Comment:
Don't know if you want to be the one volunteering for this, but I think it's about time to implement a base-class for classes that have `JSON` body, like the `mlrun.utils.db.HasStruct` class. This line along with the `full_object` property definition are simple boilerplate code that just gets repeated over and over.

Reply:
Will look into it, but not part of this PR